samples/system-test/recognize.test.js

```diff
@@ -29,8 +27,15 @@ const bucketName = `nodejs-docs-samples-test-${uuid.v4()}`;
 const cmd = `node recognize.js`;
 const cwd = path.join(__dirname, `..`);
 const filename = `audio.raw`;
+const filename1 = `Google_Gnome.wav`;
+const filename2 = `commercial_mono.wav`;
 const filepath = path.join(__dirname, `../resources/${filename}`);
+const filepath1 = path.join(__dirname, `../resources/${filename1}`);
+const filepath2 = path.join(__dirname, `../resources/${filename2}`);
 const text = `how old is the Brooklyn Bridge`;
+const text1 = `the weather outside is sunny`;
+const text2 = `Terrific. It's on the way.`;
+const text3 = `Chrome`;
 
 test.before(async () => {
   const [bucket] = await storage.createBucket(bucketName);
```

Comment:
Add `await bucket.upload(filepath1);`

Currently only filepath is uploaded. The test fails for me because `Google_Gnome.wav` hasn't been uploaded to the test bucket.

```js
test.before(async () => {
  const [bucket] = await storage.createBucket(bucketName);
  await bucket.upload(filepath);
  await bucket.upload(filepath1);
});
```

Reply:
Fixed in latest commit.